### PR TITLE
Regenerate the dev certs

### PR DESCRIPTION
**Pull request by jaym**
They're expired, this should get the tests to pass

### CODE REVIEW

**Comment by msorens** [comment]
Could you include instructions in the PR preamble above about how to do this for next time? Thanks!

File: dev/certs/event-gateway-sni-one.crt [hunk 1]
```diff
@@ -1,25 +0,0 @@
------BEGIN CERTIFICATE-----
```
Comment (stevendanna) [inline dev/certs/event-gateway-sni-one.crt:1]: Are these not needed anymore?

**Comment by stevendanna** [comment]
In general:

```
go run /src/tools/dev-cert-gen/main.go --regen-ca
go run /src/tools/dev-cert-gen/main.go --all --force
```

should do it.  Although it looks like we have a few certs that this tool doesn't know about.

**Comment by stevendanna** [comment]
Renegerated the extra certs with:

```
 go run /src/tools/dev-cert-gen/main.go event-gateway-sni-one
 go run /src/tools/dev-cert-gen/main.go event-gateway-sni-two
```

File: dev/certs/event-gateway-sni-one.crt (hunk 1 above)
Comment (jaym) [reply dev/certs/event-gateway-sni-one.crt:1]: i just emptied out the directory and reran the generate commands. maybe they dont generate everything

**Pull request merged**